src/sage/schemes/riemann_surfaces/riemann_surface.py

```diff
@@ -2193,7 +2194,7 @@ def local_N(ct, rt):
                 z_1 = a0lc.abs() * prod((cz - r).abs() - rho_z for r in a0roots)
                 n = minpoly.degree(CCzg.gen(1))
                 ai_new = [
-                    (minpoly.coefficient({CCzg.gen(1): i}))(z=cz + self._CCz.gen(0))
+                    (CCz(minpoly.coefficient({CCzg.gen(1): i})))(z=cz + self._CCz.gen(0))
```

Comment:
```suggestion
                    CCz(minpoly.coefficient({CCzg.gen(1): i}))(z=cz + self._CCz.gen(0))
```
Are these parentheses [an elegant weapon for a more civilized age](https://xkcd.com/297/)?

Reply:
as above :-)